Package managerでSDKをインポートし、SignInWithAppleの機能を利用するとエラー発生中

## 不具合の概要(Description of bug)

- SDKのバージョン(SDK version): v5.1.0
- 不具合が発生した環境(Environment requirement): Unity 2022
- 不具合を再現させる手順・サンプルコード(Step for Reproduce bug / Sample code)
  - Package managerを利用し、SDKをインポートします
  - 以下のドキュメントを参考にし、SignInWithAppleの機能の実装を行っています
    - [SignInWithAppleドキュメント](https://mbaas.nifcloud.com/doc/current/sns/apple_unity.html)   
- エラーメッセージ(Error messages)
   - Unityでコンパイルエラーが発生しています。  
![image](https://github.com/NIFCLOUD-mbaas/ncmb_unity/assets/8982421/d917b30d-fa0f-4f01-bf11-9a63388de01e)
   - エラーの詳細は以下の２件があります。
```
1) error CS1061: ‘NCMBAppleParameters’ does not contain a definition for ‘param’ and no accessible extension method ‘param’ accepting a first argument of type ‘NCMBAppleParameters’ could be found (are you missing a using directive or an assembly reference?)
2) error CS0200: Property or indexer ‘NCMBUser.AuthData’ cannot be assigned to -- it is read only
```

--- a/ncmb_unity/Assets/NCMB/Script/NCMBAppleParameters.cs
+++ b/ncmb_unity/Assets/NCMB/Script/NCMBAppleParameters.cs
@@ -25,7 +25,7 @@ namespace NCMB
     [NCMBClassName("appleParameters")]
     public class NCMBAppleParameters
     {
-        internal Dictionary<string, object> param = new Dictionary<string, object> ();
+        public Dictionary<string, object> param = new Dictionary<string, object> ();
 
         public NCMBAppleParameters(string userId, string accessToken, string clientId)
         {
@@ -45,4 +45,3 @@ namespace NCMB
         }
     }
 }
-

--- a/ncmb_unity/Assets/NCMB/Script/NCMBUser.cs
+++ b/ncmb_unity/Assets/NCMB/Script/NCMBUser.cs
@@ -91,7 +91,7 @@ namespace  NCMB
 			get {
 				return (Dictionary<string,object>)this ["authData"];
 			}
-			internal set {
+			set {
 				this ["authData"] = value;
 			}
 		}

